test/run/ch-convert.bats

```diff
@@ -493,6 +493,66 @@ test_from () {
     [[ $(stat -c %a "${out}/maxperms_file") = 777 ]]
 }
 
+@test 'ch-convert: borked xattrs' {
+    [[ -n $CH_TEST_SUDO ]] || skip 'sudo required'
+
+    cd "$BATS_TMPDIR"
+
+    borked_img="borked_image"
+    borked_file="$borked_img/home/foo"
```

Comment:
Style:

```suggestion
    borked_file="${borked_img}/home/foo"
```

Also I recommend a directory other than `/home` because that gets special treatment sometimes.